### PR TITLE
Enable Default Logo Option

### LINKED ISSUE #1981: Default Logo Action Error
**I'm submitting a ...**
- [x] bug report

**Current behaviour:**

screenshot from 2019-01-23 17-59-53

**Expected behaviour:**

**Steps to reproduce:**

**Screenshots**

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #1981 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Enable Default Logo Option

### CODE REVIEW

**Comment by sk9331657** [comment]
@abishekvashok  @mariobehling  Please review . Badgeyay Service is blocked due to this bug. Bug was originally generated from https://github.com/fossasia/badgeyay/pull/1976 .

File: frontend/app/controllers/create-badges.js [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
+import Ember from 'ember';
+const { $ } = Ember;
 const { APP } = ENV;
-
 import { inject as service } from '@ember/service';
 
 export default Controller.extend({
+  init: () => {
+    $.ajax({
+      url       : 'https://raw.githubusercontent.com/fossasia/badgeyay/development/frontend/public/images/default_logo.png',
```
Comment (abishekvashok) [inline frontend/app/controllers/create-badges.js]: Doesn't relative url work?

Comment (sk9331657) [reply frontend/app/controllers/create-badges.js]: Changed THis. Please review.

**Comment by sk9331657** [comment]
@abishekvashok  Please review.

**Comment by sk9331657** [comment]
@mariobehling  @gabru-md  Please review.

**Comment by sk9331657** [comment]
@mariobehling  @gabru-md  @yashLadha  @ParthS007  Please review and merge this.

**Comment by sk9331657** [comment]
@fossasia/badgeyay-admins  Please review

**Pull request merged**